ios/SalesforceReact/SFNetReactBridge.m

```diff
@@ -58,6 +59,9 @@ @implementation SFNetReactBridge
     NSDictionary* queryParams = [argsDict nonNullObjectForKey:kQueryParams];
     NSMutableDictionary* headerParams = [argsDict nonNullObjectForKey:kHeaderParams];
     NSDictionary* fileParams = [argsDict nonNullObjectForKey:kfileParams];
+    
+    //Set to true if boolean is absent.
+    BOOL requiresAuthentication = [argsDict nonNullObjectForKey:kRequiresAuthentication] == nil ? true: [[argsDict nonNullObjectForKey:kRequiresAuthentication] boolValue];
```

Comment:
`nil` should mean `NO`. So we should send up the opposite (`notAuthenticated`). See my comment below.